packages/databyss-ui/modules/Landing/Landing.js

```diff
@@ -1,48 +1,128 @@
+import LandingBody from '../../components/Viewport/LandingBody'
 import Raw from '../../components/Viewport/Raw'
 import PageHeading from '../../components/Heading/PageHeading'
 import PageSubHeading from '../../components/Heading/PageSubHeading'
 import PageNav from '../../components/Navigation/PageNav'
 import CommaSeparatedList from '../../components/List/CommaSeparatedList'
+import LandingHeading from '../../components/Heading/LandingHeading'
 import ContentHeading from '../../components/Heading/ContentHeading'
 import styles from './styles'
 
-const Landing = ({
-  className,
-  classes,
-  cfList,
-  renderCfItem,
-  contentTitle,
-  children,
-  title,
-  subtitle,
-}) => (
-  <Content className={classnames(className, classes.landing)}>
-    <PageHeading>
-      <Raw html={title} />
-    </PageHeading>
-    {subtitle && (
-      <PageSubHeading>
-        <Raw html={subtitle} />
-      </PageSubHeading>
-    )}
-    {cfList && (
-      <PageNav ariaLabel="compare with">
-        <CommaSeparatedList opener="[cf." closer="]">
-          {cfList.map(cf =>
-            React.cloneElement(renderCfItem(cf), { key: cf.id })
+class Landing extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      borderBottom: false,
+      contentLoaded: false,
+      headerLoaded: false,
+    }
+  }
+
+  componentDidMount() {
+    window.addEventListener('scroll', this.handleScroll.bind(this), true)
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('scroll', this.handleScroll.bind(this), true)
+  }
+
+  handleScroll() {
+    // check to see both header and content header are mounted
+    const { contentLoaded, headerLoaded, borderBottom } = this.state
+    if (document.getElementById('landing-header') && !headerLoaded) {
+      this.setState({ headerLoaded: true })
+    }
+    if (document.getElementById('content-header') && !contentLoaded) {
+      this.setState({ contentLoaded: true })
+    }
+
+    // if both elements are loaded apply logic to set the bottomBorder state
+    if (contentLoaded && headerLoaded) {
+      const headerBottom = document
+        .getElementById('landing-header')
+        .getBoundingClientRect().bottom
+      const contentHeaderTop = document
+        .getElementById('content-header')
+        .getBoundingClientRect().top
+
+      if (headerBottom > contentHeaderTop && !borderBottom) {
+        this.setState({ borderBottom: true })
+      }
+
+      if (headerBottom < contentHeaderTop && borderBottom) {
+        this.setState({ borderBottom: false })
+      }
+    }
+  }
+
+  render() {
+    const {
+      className,
+      classes,
+      cfList,
+      renderCfItem,
+      contentTitle,
+      children,
+      title,
+      subtitle,
+      showMotifLinks,
+      onMotifLinksChange,
+      withToggle,
+    } = this.props
+
+    return (
+      <Content className={classnames(className, classes.landing)}>
+        <LandingHeading
+          id="landing-header"
```

Comment:
rather than use a DOM id (which is static and will fail if more than one `<Landing />` is on a page), use `ref={this.headerRef}` (see comment in constructor)

Reply:
for future reference `react-jss` breaks `React.forwardRef` because it removes the `ref` component. Found a solution here [here](https://github.com/cssinjs/react-jss/issues/277)